app/tests/api/organization-model.test.js

```diff
@@ -0,0 +1,222 @@
+const path = require('path')
+const test = require('ava')
+const { prop, map, contains } = require('ramda')
+const db = require('../../db')
+const organization = require('../../lib/organization')
+const team = require('../../lib/team')
+
+const migrationsDirectory = path.join(__dirname, '..', '..', 'db', 'migrations')
+
+test.before(async () => {
+  const conn = await db()
+  await conn.migrate.latest({ directory: migrationsDirectory })
+
+  // seed
+  await conn('users').insert({ id: 1 })
+  await conn('users').insert({ id: 2 })
+  await conn('users').insert({ id: 3 })
+  await conn('users').insert({ id: 4 })
+})
+
+test.after.always(async () => {
+  const conn = await db()
+  await conn.migrate.rollback({ directory: migrationsDirectory })
+  conn.destroy()
+})
+
+/**
+ * Test organization creation
+ * An organization is created by a user.
+ * The user becomes the owner of that organization and a manager of that organization
+ */
+test('create an organization', async (t) => {
+  // setup
+  const name = 'create test'
+  const user = 1
+  const data = await organization.create({ name }, user)
+
+  // tests
+  t.is(data.name, name)
+  t.truthy(data.name, name)
```

Comment:
this call to `truthy` seems redundant and could be removed.

Reply:
```suggestion
```